Scroll To Top Feature

pelonchasva opened the pull request:
These changes were made in order to add the scroll to top feature in item #14.

It adds a specific scss definition for the html element used in the Store page and adds the window scroll event to a custom function to handle the scroll behavior.

On the base.scss file, a new entry was added to change the scroll-behavior property to smooth, so the transition is more pleasant to the user.

MehediH commented:
Hey, this looks great -- thank you! Just some changes I'd like to request before I merge this in, though:

* It would be a better idea to move the component inside the SelecitonBar component, or even the Footer so that it shows up on all pages. This is important as we need the functionality on the homepage too.
* Styling-wise, I would move the button to be on top of the selection bar at the bottom or at least be aligned on the center (vertically).
* And lastly, it would be sweet to have the button show up more smoothly (maybe a little CSS fade-in animation) 


Thanks!

pelonchasva commented:
Hey @MehediH ,

Based on your comments, I moved the component into the Footer component so it is available application-wide.

I fixed the alignment of the button to the center when the SelectionBar shows up.

I added the fade-in property to the button, and since I'm not that good with CSS, I wanted to add the fade-out property too, but that is just out of my knowledge so far, so apologies for that.

Are these changes better?

MehediH commented:
Perfect -- thank you!
